Makefile

```diff
@@ -40,9 +40,10 @@ coverage.go:
 docs.generate:
 	# Temporarily replaces ~/.rill/config.yaml to avoid including user-defined defaults in generated docs.
 	# Sets version to the latest tag to simulate a production build, where certain commands are hidden.
-	rm -rf docs/docs/reference/cli
+	rm -rf docs/docs/reference/cli docs/docs/reference/project-files
```

Comment:
We don't want to delete the old manually maintained files just yet. Can we output to a separate directory in this PR? For example, could be `docs/docs/reference/yaml` or something like that.

And then when everyone's happy with the new docs, we can do a separate PR that does the full switch.